tests/data/test_btanalysis.py

```diff
@@ -362,3 +362,38 @@ def test_calculate_max_drawdown2():
     df = DataFrame(zip(values[:5], dates[:5]), columns=['profit', 'open_date'])
     with pytest.raises(ValueError, match='No losing trade, therefore no drawdown.'):
         calculate_max_drawdown(df, date_col='open_date', value_col='profit')
+
+
+@pytest.mark.parametrize('profits,relative,highd,lowd,result,result_rel', [
+    ([0.0, -500.0, 500.0, 10000.0, -1000.0], False, 3, 4, 1000.0, 0.090909),
+    ([0.0, -500.0, 500.0, 10000.0, -1000.0], True, 0, 1, 1000.0, 0.5),
```

Comment:
I've added a test case for the 2 scenarios we are looking at - with the (very simple) test numbers, and the result looks quite promising.

The thing that surprised me here is that absolute drawdown remains 1000 - even though "relative" is used.
In my understanding, all returned values should align - so if i use `relative` - i get "high/low" from the initial drawdown (the biggest one relative wise, relative to balance) - including dates, and absolute drawdown also from this period.

currently, we get the 500$ drawdown (for all values _except_ the absolute drawdown - which is quite odd - as all other values are from the initial drawdown period.

Reply:
Yes you are right, absolute drawdown result was not matching the relative result when the `relative` flag was being used, I just added a commit to fix that